pontoon/base/static/js/translate.js

```diff
@@ -2045,7 +2045,9 @@ var Pontoon = (function (my) {
 
                   // Last translation deleted, no alternative available
                   } else {
-                    $('#clear').click();
+                    if (self.cachedTranslation === $('#translation').val()) {
+                      $('#clear').click();
+                    }
```

Comment:
What happens here is now is that we check if the text in the translation input is different than the last active translations. If yes, we don't clear the input.